index.html

```diff
@@ -1,84 +1,34 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
 <head>
-    <meta charset="utf-8" />
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Scratch Clone - CS141</title>
     <script src="static/jquery-3.2.1.min.js"></script>
     <script src="static/google-blockly/blockly_compressed.js"></script>
     <script src="static/google-blockly/blocks_compressed.js"></script>
     <script src="static/google-blockly/msg/js/en.js"></script>
-    <style>
-        body {
-            padding: 0;
-            margin: 0;
-            height: 100%;
-            display: flex;
-            flex-direction: column;
-        }
-
-        #controls {
-            z-index: 999;
-            padding: 20px;
-            background-color: #2c1e37;
-            box-shadow:
-                rgba(0, 0, 0, 0.137255) 0px 0px 4px 0px,
-                rgba(0, 0, 0, 0.278431) 0px 4px 8px 0px;
-            text-align: right;
-        }
-
-        #editor {
-            flex: 1;
-            display: flex;
-            flex-direction: row;
-        }
-
-        #compiledCode {
-            border-bottom: 1px solid #c6c6c6;
-        }
-
-        #output {
-            overflow-y: scroll;
-            height: 200px;
-            font-family: sans-serif;
-        }
-
-        #output .message {
-            width: 100%;
-            padding-top: 50px;
-            text-align: center;
-        }
-
-        #output .variable {
-            display: flex;
-            flex-direction: row;
-
-            font-size: 20pt;
-            border-bottom: 1px solid black;
-        }
-
-        #output .variable div {
-            flex: 1;
-            padding: 20px;
-        }
-
-        #output .variable .name {
-            border-right: 1px solid black;
-        }
-    </style>
+    <script src="static/FileSaver.js"></script>
+    <link rel="stylesheet" type="text/css" href="static/index.css">
 </head>
-<body style="padding: 0; margin: 0;">
-    <div id="controls">
+<body>
+    <header>
+        <span id="importExportBox">
+            <input id="importInput" type="file">
+            <button id="importButton"></button>
```

Comment:
Was removing the "Input" text here intentional?
```suggestion
            <button id="importButton">Import</button>
```